Protecs/src/test/java/uk/ac/ucl/protecs/objects/diseases/CoronavirusInfectiousBehaviourTesting.java

```diff
@@ -26,8 +24,7 @@ public class CoronavirusInfectiousBehaviourTesting {
 	@Test
 	public void ifThereAreNoCovidInfectionsPeopleStaySusceptible() {
 		// create a simulation and start
-
-		WorldBankCovid19Sim sim = helperFunctions.CreateDummySim("src/main/resources/InfectiousBehaviourTestParams.txt");
+		WorldBankCovid19Sim sim = helperFunctions.CreateDummySim("src/test/resources/InfectiousBehaviourTestParams.txt");
```

Comment:
let's refactor this to make the directory name a variable so we can change it more easily in the future!